### DISCUSSION TITLE
YouTube videos showing in visual editor edit mode, but not in page view

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#5256] [Error / Bug Report]
Hi,

Want to ask about this problem.

I know that you can fix it by injecting code (as seen here in [1459#issuecomment-889564668](https://github.com/requarks/wiki/issues/1459#issuecomment-889564668) but I was wondering if devs will fix this? Maybe in version 3?

Thank you.

**Edit:** apparently that thread was deleted, I took it from [discussions/5043](https://github.com/requarks/wiki/discussions/5043).

**Comment by asisteclick** [comment]
Hi! I'm having the same issue, Youtube videos display on editing mode but not on the page itself seen as a final user. 

Did you have any solution to this?

Thanks!
